sale_force_cancel_cft/views/sale_view.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+  <data>
+
+    <record model="ir.ui.view" id="view_order_form_force_cancel">
+      <field name="name">sale.order.force.cancel</field>
+      <field name="model">sale.order</field>
+      <field name="inherit_id" ref="sale.view_order_form"/>
+      <field name="arch" type="xml">
+        <button name="action_cancel" position="after">
+          <button name="force_cancel" states="manual,progress" string="Force Cancel" type="object" groups="sale_force_cancel_cft.group_force_cancel_so"/>
```

Comment:
As we discuss, we should use workflow. Please change type="workflow" and enhance the existing workflow.